__tests__/app.test.js

```diff
@@ -126,7 +125,7 @@ describe('/api/articles/:article_id/comments', () => {
                         created_at: expect.any(String),
                         author: expect.any(String),
                         body: expect.any(String),
-                        article_id: expect.any(Number)
+                        article_id: 9
```

Comment:
Please only make changes relevant to the current ticket you are working on. Any changes to past tickets should be made on those branches or a new/different branch to the current ticket you are working on.